pygmo/_py_algorithms.py

```diff
@@ -0,0 +1,131 @@
+import random
+
+try:
+    from scipy.optimize import minimize
+
+    class scipy:
+        """
+        This class is a user defined algorithm (UDA) providing a wrapper around the function scipy.optimize.minimize.
+        The constructor accepts those arguments that are specific to the algorithm:
```

Comment:
The constructor accepts those arguments that are specific to the algorithm:
->
Only the following arguments specific to the scipy algorithm are accepted upon construction:

Add the documentation on what happens if these are passed wrongly

Reply:
Adapted Docstring.